perl_mongers.xml

```diff
@@ -410,14 +410,14 @@
       <latitude>35.19755</latitude>
     </location>
     <tsar>
-      <name>William McKee</name>
+      <name>Jason Crome</name>
       <email type="personal">
-        <user>william</user>
-        <domain>knowmad.com</domain>
+        <user>cromedome</user>
+        <domain>gmail.com</domain>
       </email>
     </tsar>
-    <web>http://charlotte.pm.org/</web>
-    <date type="inception">19990127</date>
+    <web>https://www.meetup.com/charlotte-pm/</web>
+    <date type="inception">20190101</date>
```

Comment:
We usually leave the inception date set to the original (oldest) iteration of a group, if that's cool with you. :)

Reply:
That works for me!